### PR TITLE
[IMP] show incoming date of quant, easier to pick

### PR DESCRIPTION
**Pull request by fanha99**

### CODE REVIEW

File: stock_quant_manual_assign/wizard/assign_manual_quants.py [hunk 1]
```diff
@@ -109,6 +110,8 @@ def onchange_selected(self):
         comodel_name='stock.quant.package', string='Package',
         related='quant.package_id', readonly=True,
         groups="stock.group_tracking_lot")
+    in_date = fields.Date(
+        string='In Date', readonly=True)
```
Comment (rousseldenis) [inline stock_quant_manual_assign/wizard/assign_manual_quants.py]: @fanha99 Maybe 'Incoming Date' should be more explicit ?

**Review by rousseldenis — CHANGES_REQUESTED** [review]
Minor comment but :+1:

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Please change version number according: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#version-numbers

**Review by pedrobaeza — APPROVED** [review]

**Review by rousseldenis — APPROVED** [review]
:+1: Code review

**Pull request merged**